packages/devextreme/js/ui/list/ui.list.edit.js

```diff
@@ -357,6 +357,7 @@ const ListEdit = ListBase.inherit({
         const editStrategy = this._editStrategy;
         const deletingElementIndex = editStrategy.getNormalizedIndex(itemElement);
         const focusedElement = this.option('focusedElement');
+        const shouldBeFocused = this.option('focusStateEnabled');
```

Comment:
When we just get an option value it's better no not change its name. Then in the code where we use it we can explicitly see from what option the behavior is dependent.
```suggestion
        const focusStateEnabled = this.option('focusStateEnabled');
```

Reply:
[7899758](https://github.com/DevExpress/DevExtreme/pull/27230/commits/78997589a5ee5de920c419bd20c70dafc6adba90)